doc/source/admin/scaling.md

```diff
@@ -400,6 +400,20 @@ If you omit the `processes` argument this will default to a single process.
 You can further customize the handler names using the `name_template` section,
 for a complete example see [this gravity test case](https://github.com/galaxyproject/gravity/blob/a00df1c671fdc01e3fbc42f64a851a45a37a1870/tests/test_process_manager.py#L28).
 
+You can define arbitrary environment variables (e.g. ``$DRMAA_LIBRARY_PATH``) for dynamic handlers using the
+``environnment`` key on a handler definition:
+
+```yaml
+gravity:
+  handlers:
+    handler:
+      processes: 3
+      pools:
+        - job-handlers
+      environment:
+        DRMAA_LIBRARY_PATH: /usr/lib/libdrmaa.so
```

Comment:
I feel this is better set at the plugin level, as you also mentioned in https://github.com/galaxyproject/gravity/issues/52#issuecomment-1158027779 . Not sure we want to encourage this?

Reply:
This is just an example, I can use a different variable. The other common use case I can think of off the top of my head is setting `$PYTHONPATH` for dynamic job runner rules. Or I can just use a completely unrelated example like `FOO=bar`.